Sources/swift-format/Utilities/Diagnostic.swift

```diff
@@ -47,6 +47,12 @@ struct Diagnostic {
       self.line = findingLocation.line
       self.column = findingLocation.column
     }
+
+    init(file: String, line: Int, column: Int) {
```

Comment:
It seems like before we only made `Diagnostic.Location` out of `SourceLocation`, but I did not want to import `SwiftSyntax` into `swift-format` target. Cool to use it directly?